src/chrome/content/rules/QuiteRSS.org.xml

```diff
@@ -1,14 +1,15 @@
 <!--
-Disabled by https-everywhere-checker because:
+Pre-2015-09-18 Disabled by https-everywhere-checker because:
 Fetch error: http://www.quiterss.org/ => https://www.quiterss.org/: (51, "SSL: no alternative certificate subject name matches target host name 'www.quiterss.org'")
+2015-09-18 Remove default_off since website has been update to fix the error and update to new standard.
 -->
-<ruleset name="QuiteRSS.org" default_off='failed ruleset test'>
+<ruleset name="QuiteRSS.org"></ruleset>
```

Comment:
```</ruleset>``` should be removed. Think of it as an array, ```<ruleset name="QuiteRSS.org"></ruleset>``` is simply declaring an empty ruleset. Instead, what closes the opened tag is the ```</ruleset>``` at the bottom.